x-pack/legacy/plugins/snapshot_restore/public/app/sections/policy_edit/policy_edit.tsx

```diff
@@ -182,17 +183,35 @@ export const PolicyEdit: React.FunctionComponent<RouteComponentProps<MatchParams
     }
 
     return (
-      <PolicyForm
-        policy={policy}
-        indices={indices}
-        currentUrl={pathname}
-        isEditing={true}
-        isSaving={isSaving}
-        saveError={renderSaveError()}
-        clearSaveError={clearSaveError}
-        onSave={onSave}
-        onCancel={onCancel}
-      />
+      <>
```

Comment:
Can remove this outer fragment.

Reply:
I think this one is still needed as well